test/test_exception_handling.py

```diff
@@ -0,0 +1,31 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from wampy.errors import RemoteError
+from wampy.peers.clients import Client
+from wampy.roles.callee import callee
+
+
+class UnreliableCallee(Client):
+
+    @callee
+    def get_foo(self, *args, **kwargs):
+        raise ValueError(
+            "i don't like any of your values: %s, %s", args, kwargs
+        )
+
+
+@pytest.yield_fixture
```

Comment:
check with pytest, yield_fixture decorator is probably obsolete, you can use fixture instead